### PR TITLE
doitlive: 3.0.3 -> 4.0.1

### PR DESCRIPTION
**Pull request by mbode**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
```
@GrahamcOfBorg build doitlive python27Packages.click-completion python27Packages.click-didyoumean python27Packages.shellingham python36Packages.click-completion python36Packages.click-didyoumean python36Packages.shellingham
```

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -554,6 +554,8 @@ in {
 
   serversyncstorage = callPackage ../development/python-modules/serversyncstorage {};
 
+  shellingham = callPackage ../development/python-modules/shellingham {};
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Can you put those into individual commits?

Comment (mbode) [reply pkgs/top-level/python-packages.nix]: done, also disabled the failing python2 build

**Comment by Ekleog** [comment]
@GrahamcOfBorg build doitlive python27Packages.click-completion python27Packages.click-didyoumean python27Packages.shellingham python36Packages.click-completion python36Packages.click-didyoumean python36Packages.shellingham

**Comment by Ekleog** [comment]
(from triage) Looks ready to me, though I can't call the tests on darwin.

**Pull request merged**